src/volume_calc.cpp

```diff
@@ -125,15 +125,16 @@ std::vector<VolumeCalculation::Result> VolumeCalculation::execute() const
       std::vector<std::vector<int>> hits(n);
       Particle p;
 
-      prn_set_stream(STREAM_VOLUME);
+      uint64_t prn_seeds[N_STREAMS];
+      int stream = STREAM_VOLUME;
 
       // Sample locations and count hits
       #pragma omp for
       for (size_t i = i_start; i < i_end; i++) {
-        set_particle_seed(iterations * n_samples_ + i);
+        set_particle_seed(iterations * n_samples_ + i, prn_seeds);
```

Comment:
So I'm thinking that in these cases where we are outside of the transport loop, we really should only be creating a single seed rather than having the array and only using one seed from it. This implies that `set_particle_seed` should probably work with just a single seed rather than requiring the full array.